src/containers/Aya/AyaContact/AyaContact.scss

```diff
@@ -13,4 +13,10 @@
 .emailAddress {
     margin-left: 5px;
 }
+ul {
```

Comment:
Does this have any effect, its not inside a style?

Reply:
No.  Good spot, I have removed it.